ui/sign_in_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+describe 'Sign in page' do
+  context 'when unauthenticated' do
```

Comment:
I'd argue we could do with out this context if we aren't also checking the behaviour of an authenticated user.